### DISCUSSION TITLE
SimpleFIN Bridge integration

### DISCUSSION THREAD

**Discussion by darkknight7777777** [ghostfolio/ghostfolio#3424] [Ideas]
For US brokers it would be nice to be able to sync data using [simplefin](https://beta-bridge.simplefin.org/info/developers). There are already hundreds of supported institutions, it would just need to be integrated into ghostfolio.

**Comment by dtslvr** [comment]
Hi @darkknight7777777

Maybe you can get inspiration from similar projects by the community [here](https://github.com/topics/ghostfolio) and write your own script to sync data using _SimpleFIN Bridge_ and Ghostfolio's Public API.
